bots/mailman/bot.js

```diff
@@ -1,44 +1,91 @@
+function locationsSuggestions (params) {
+    var result = {title: "Send location"};
+    var seqArg = params.seqArg ? params.seqArg : "";
+
+    if (seqArg == "Dropped pin")
```

Comment:
not super important, but: https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons